Source/WebCore/platform/graphics/GraphicsContextGL.cpp

```diff
@@ -425,118 +425,62 @@ unsigned GraphicsContextGL::computeBytesPerGroup(GCGLenum format, GCGLenum type)
     }
 }
 
-GCGLenum GraphicsContextGL::computeImageSizeInBytes(GCGLenum format, GCGLenum type, GCGLsizei width, GCGLsizei height, GCGLsizei depth, const PixelStoreParameters& params, unsigned* imageSizeInBytes, unsigned* paddingInBytes, unsigned* skipSizeInBytes)
+std::optional<GraphicsContextGL::PixelRectangleSizes> GraphicsContextGL::computeImageSize(GCGLenum format, GCGLenum type, IntSize size, GCGLsizei inDepth, const PixelStoreParameters& parameters)
```

Comment:
I'm not a fan of `inDepth`, but I can't think of a better name. Does WebKit have a 3D size type?

Reply:
> Does WebKit have a 3D size type?

No.